server/zanata-frontend/src/frontend/app/editor/containers/Sidebar/index.css

```diff
@@ -380,6 +380,17 @@ h2.validation {
   color: var(--Sidebar-color-text);
 }
 
+.settings-options h2 {
```

Comment:
Shall remove this css - was for headings which wont be included in this component